qa/suites/fs/libcephfs/overrides/multimds.yaml

```diff
@@ -0,0 +1,4 @@
+overrides:
+  ceph:
+    cephfs:
+      max_mds: 2
```

Comment:
Probably also turn off the automatic balancer so we can be sure the other libcephfs tests do not break in strange ways:

`ceph fs set cephfs balance_automate false`

That's the default but it should be explicit in these tests. (You can also make this a config in the overrides which is applied by `filesystem.py`.)